service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2087,6 +2087,9 @@ components:
       properties:
         sql:
           type: string
+        entityIdSql:
```

Comment:
We would like to add this field to the describeExport API call. We are currently using this endpoint cause the [response](https://github.com/DataBiosphere/tanagra/blob/9f049c438af6002d25636c16503287e45487e636/service/src/main/resources/api/service_openapi.yaml#L2825-L2861) provides : 

- indexSql - if source tables aren't configured for this entity
- sourceSql - if source tables are configured for this entity

We would like to add entityIdSql to this response.

Reply:
sounds good changed